Tribler/Test/Core/Upgrade/upgrade_base.py

```diff
@@ -0,0 +1,74 @@
+import os
+import shutil
+import Tribler
+from Tribler.Core.CacheDB.sqlitecachedb import SQLiteCacheDB, DB_SCRIPT_NAME
+from Tribler.Core.Session import Session
+from Tribler.Core.SessionConfig import SessionStartupConfig
+from Tribler.Test.Core.base_test import TriblerCoreTest
+
+
+class MockTorrentStore(object):
+            pass
+
+
+class AbstractUpgrader(TriblerCoreTest):
+
+    FILE_DIR = os.path.abspath(os.path.dirname(os.path.realpath(__file__)))
+    DATABASES_DIR = os.path.abspath(os.path.join(FILE_DIR, u"../data/upgrade_databases/"))
+
+    def write_data_to_file(self, file_name):
+        with open(file_name, 'w') as file:
+            file.write("lorem ipsum")
+            file.close()
+
+    def setUpPreSession(self):
```

Comment:
Ah apparently I don't need the `setUpPreSession` method at all since I'm only creating a `Session` object, not starting it. I will remove the method.